infra/core/app-configuration/config-data/config-data.json

```diff
@@ -0,0 +1,86 @@
+[
+  {
+    "key": "ENVIRONMENT",
+    "value": "#{{ lower(environment) }}",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "APPLICATIONS_ACR_NAME",
+    "value": "#{{ infraResourceNamePrefix }}#{{ nc_resource_containerregistry }}#{{ nc_instance_regionid }}01",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "FLUX_CONFIG_NAMESPACE",
+    "value": "flux-config",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "SERVICEBUS_RG",
+    "value": "#{{ servicesResourceGroup }}",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "SERVICEBUS_NS",
+    "value": "#{{ infraResourceNamePrefix }}#{{ nc_resource_servicebus }}#{{ nc_instance_regionid }}01",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "SHRD_POSTGRES_SERVER_RG",
+    "value": "#{{ dbsResourceGroup }}",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "SHRD_POSTGRES_SERVER_#{{ nc_instance_regionid }}01",
```

Comment:
why do we have region ID in here? should be a static key with a dynamic value no?

Reply:
Removed Region Id, Key Should be static